### PR TITLE
chore(ui): drop the Ropsten testnet

### PR DESCRIPTION
**Pull request by rudolfs**
We're well served by `Rinkeby` and don't want to spend time unnecessarily on maintaining multiple test nets.
See individual commits for the doc changes.

### CODE REVIEW

File: ETHEREUM.md [hunk 1]
```diff
@@ -57,12 +56,8 @@ Read more about Orgs [here][org].
...
   real-world conditions.
 
-  You will need fake Rinkeby Eth (aka `rEth`) to pay the incurring transactions
-  fees.  No real money will be charged.
-
-  As in the `Local` environment, here the "DAI" token used is also our custom
-  Radicle Token, deployed in the Rinkeby network. Reach out to the team to get
-  some Radicle Tokens into your account.
+  You will need [Rinkeby Eth] to pay the incurring transactions fees. No real
+  money will be charged.
```
Comment (geigerzaehler) [inline ETHEREUM.md:60]:
```suggestion
  money will be used.
```

File: ETHEREUM.md [hunk 2]
```diff
@@ -16,60 +13,65 @@ incoming support funds, and others, translate into ethereum transactions.
...
-1. `Local`
+Radicle Orgs allow a group of people to manage multiple projects on Radicle.
 
-  In this environment, the Radicle Contracts are deployed to development Ethereum
-  node running locally and we use a local test wallet that can communicate with it.
-  The "DAI" token used in this environment is our custom Radicle Token. See the
-  [local environment](#local-environment) section to learn how to get set up.
+What’s cool about Orgs is that they’re actually smart contracts that can be
+managed by DAOs or multi-sigs. With Radicle Orgs, projects can maintain an
+auditable and transparent history of project state on-chain. This enables new
```
Comment (geigerzaehler) [inline ETHEREUM.md:35]: A bit less editorializing ;) and swapping the what and how:

> Radicle orgs allow users to maintain an auditable and transparent history of project state on Ethereum. [...] Org contracts may be controlled by arbitrary addresses. At the moment we deploy a Gnosis safe contract (link) as the controller when creating an org.

Comment (rudolfs) [reply ETHEREUM.md:35]: 👍 I just copy-pasted it straight from https://radicle.community/t/feature-update-orgs/2132/4.

Comment (rudolfs) [reply ETHEREUM.md]: c87333c01e30930a8771878ac5f552e3bb4609f7

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**